source/2019-12-20-the-ember-times-issue-129.md

```diff
@@ -0,0 +1,159 @@
+---
+title: The Ember Times - Issue No. 129
+author: Chris Ng, Amy Lam, Isaac Lee, Godfrey Chan, Jessica Jordan
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
+alias : "blog/2019/12/20-the-ember-times-issue-129.html"
+responsive: true
+---
+
+Happy Holidays, Emberistas! 🐹🌲⛄️
+
+Ember Octane is here 🐹🎉,
+new ember-autofocus-modifier 💡,
+3 new RFCs for template helpers 3️⃣,
+ember-apollo-client v2 released 🎉
+and join us for the last 11 days of DecEmber 1️⃣️⃣1️⃣!
```

Comment:
```suggestion
and join us for the last 11 days of DecEmber 1️⃣️1️⃣!
```

Reply:
Good catch, I updated it in a separate commit as well :+1: